integreat_cms/core/settings.py

```diff
@@ -697,7 +697,7 @@
 
 #: A list of directories where Django looks for translation files
 #: (see :setting:`django:LOCALE_PATHS` and :doc:`django:topics/i18n/index`)
-LOCALE_PATHS = (os.path.join(BASE_DIR, "locale"),)
+LOCALE_PATHS = (os.path.join(BASE_DIR, "locale"), "/etc/integreat/locale")
```

Comment:
I'm wondering if we should make this path configurable via the integreat-cms.ini file.?

Reply:
Yes, why not :+1:
The `/etc/integreat/locale` can still be used as default value when it's not overridden by the config file.

Reply:
Indeed. One relevant note: it is okay for a path to not exist. Django just ignores missing directories.